app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -166,6 +168,9 @@ const ListingDetail = () => {
                 <ListingDetailsWaitlist listing={listing} />
               </Mobile>
             )}
+            {isFcfsSalesListing(listing) && isSalesFcfsEnabled && (
+              <Mobile>{fcfsNoLotteryRequired()}</Mobile>
```

Comment:
Not a blocker, but curious as to why you did this as a function and not as a component?